server/graphql-engine.cabal

```diff
@@ -293,6 +294,7 @@ common lib-depends
                      , some
                      , split
                      , template-haskell
+                     , ghc-heap-view
```

Comment:
I assume this change is just for testing.

Reply:
your CPP module was essentially upstreamed into the latest ghc-heap-view version. The surrounding code failed to compile until these changes were made. Of course the runtime dependency for this should be made optional, but Nix did not seem to find it using the conditional profiling include.

Reply:
@jberryman: Do you think we can upgrade `ghc-heap-view` to take advantage of this?